tests/e2e/tests/build/scripts-array.ts

```diff
@@ -48,5 +50,7 @@ export default function () {
       <script type="text/javascript" src="scripts.bundle.js"></script>
       <script type="text/javascript" src="vendor.bundle.js"></script>
       <script type="text/javascript" src="main.bundle.js"></script>
-    `));
+    `))
+     // ensure scripts aren't using script-loader when imported from the app
+    .then(() => expectFileToMatch('dist/main.bundle.js', 'console.log(\'string-script\');'));
```

Comment:
script-loader also causes CSP issues since it uses `eval` so removing it would be good in general.

Could the scripts be processed (concat, etc.) beforehand and then added as a webpack asset.  or maybe a more extensive plugin or loader?